server/midddleware/forwarder.go

```diff
@@ -24,6 +30,35 @@ type Forwarder struct {
 
 var forwarder = Forwarder{}
 
+func requestToResponse(method string) (proto.Message, proto.Message) {
+	log.Debug().Str("method", method).Msg("requestToResponse")
+	switch strings.TrimPrefix(method, "/tigrisdata.v1.Tigris/") {
+	case "Insert":
```

Comment:
I think we have this defined somewhere?

Reply:
I've tried to look it up. Let me try once again.

Reply:
Should we define it? To avoid it getting out of sync.

Reply:
Are we talking about the prefix stored as a constant somewhere? If so, I agree.

Reply:
Yeah, prefix and method names. We'll define constants in later diff, currently it used here only.